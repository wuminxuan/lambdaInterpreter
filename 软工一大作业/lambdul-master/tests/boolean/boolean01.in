\x.(\y.(y))
\x.(\y.(x))
((\f.(\x.(\y.((f y) x)))) \x.(\y.(y)))
((\f.(\x.(\y.((f y) x)))) ((\f.(\x.(\y.((f y) x)))) (\x.(\y.(y)))))
((\f.(\x.(\y.((f y) x)))) \x.(\y.(x)))
((\f.(\x.(\y.((f y) x)))) ((\f.(\x.(\y.((f y) x)))) (\x.(\y.(x)))))
(\f.(\x.(\y.((f y) x))))
