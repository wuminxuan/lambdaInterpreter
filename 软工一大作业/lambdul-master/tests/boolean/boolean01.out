λx.(λy.(y))
λx.(λy.(x))
λx.(λy.(x))
λx.(λy.(y))
λx.(λy.(y))
λx.(λy.(x))
λf.(λx.(λy.(((f y) x))))
